src/Tools/GenerateAnalyzerNuspec/Program.cs

```diff
@@ -1,5 +1,17 @@
-using System.Diagnostics;
+#pragma warning disable CA1820, CS8600, IDE0055, IDE0057, IDE0062, IDE0078, IDE0073
```

Comment:
Can you please file a tracking issue to fix these suppressions in a follow-up change?

Reply:
I filed https://github.com/dotnet/roslyn-analyzers/issues/4441